mozilla/foundation.mozilla.org
# child safety blurb field added to product page

Closes #8506 
Related PRs/issues #

**Link to sample test page**:

[Sample Page with Child Privacy Information](https://foundation-s-pni-child--xmwl3y.mofostaging.net/en/privacynotincluded/under-opportunity-middle-region-prevent-stand-hold-range/)
[Sample page without Child Privacy Information](https://foundation-s-pni-child--xmwl3y.mofostaging.net/en/privacynotincluded/mother-become-sell/)

**Steps to test**:

1. Please visit the review pages above
2. The first link is a page with Child Privacy information, so the field should render on the product page underneath the track record section.
3. The 2nd link is a page where Child Privacy information is not set, so it should *not* render on the products page.
4. If you would like to see what this field looks like in the CMS, please log in using username and password (review/review)
5. If everything is looking as expected, testing is complete!
